src/components/Loading Screen/LoadingScreen.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
```

Comment:
The folder name contains a whitespace. I think this is an mistake.

Reply:
Thank you i changed the folder's name.